packages/dashboard/middleware/sidetree.ts

```diff
@@ -11,6 +12,7 @@ const methods: any = {
   // https://ethereum.stackexchange.com/questions/111540/cant-resolve-electron-in-opt-build-repo-node-modules-swarm-js-node-modules
   'elem:ganache': element,
   'elem:ropsten': element,
+  photon,
```

Comment:
we have do qldb ledger for test, it is called `photon-test` and we have one for production called `photon`, I thought it made sense to call it just photon since it is that production one.  I could create another for `photon:testnet` that points to `photon-test` qldb ledger as well if that makes sense.